FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1641,12 +1703,12 @@ namespace edm {
       rng->postEventRead(ev);
     }
 
-    WaitingTaskHolder finalizeEventTask(
-        make_waiting_task(tbb::task::allocate_root(), [this, pep, iHolder](std::exception_ptr const* iPtr) mutable {
+    WaitingTaskHolder finalizeEventTask(make_waiting_task(
+        tbb::task::allocate_root(), [this, pep, iHolder, iStreamIndex](std::exception_ptr const* iPtr) mutable {
           //NOTE: If we have a looper we only have one Stream
```

Comment:
Here was the comment about streams and loopers

Reply:
OK.